lib/gds_api/email_alert_api.rb

```diff
@@ -110,7 +110,7 @@ def unsubscribe(uuid)
   #
   # @return [nil]
   def unsubscribe_subscriber(id)
-    delete_json("#{endpoint}/subscribers/#{id}")
+    delete_json("#{endpoint}/subscribers/#{uri_encode(id)}")
```

Comment:
From the database, it looks like this will always be a number, which won't need encoding?